.github/workflows/gamescope-session.yml

```diff
@@ -29,19 +21,14 @@ jobs:
       - name: Copy interal source files
```

Comment:
Minor typo `internal`

Reply:
Thanks! I need to use spell check more often. :-)